src/lib/mina/actions/offchain-contract-tests/single-contract-instance.unit-test.ts

```diff
@@ -0,0 +1,62 @@
+import { UInt64 } from '../../../../index.js';
+import * as Mina from '../../mina.js';
+import assert from 'assert';
+
+import {
+  ExampleContract,
+  offchainState as exampleOffchainState,
+} from './ExampleContract.js';
+import { settle, transfer } from './utils.js';
+
+const Local = await Mina.LocalBlockchain({ proofsEnabled: true });
```

Comment:
Proofs enabled on the shorter test to make sure the artifacts compile correctly and proofs can generate.